hardware/README.md

```diff
@@ -1,6 +1,6 @@
 # ssis:bit hardware
 
-The foundation of the ssis:bit is a [LilyGo T8 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) board with [CircuitPython](https://circuitpython.org/). It has already many key features out of the box:
+The foundation of the ssis:bit is a [LilyGo T8 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) board (in GitHub as [T-Display S2](https://github.com/Xinyuan-LilyGO/LilyGo-T-Display-S2)) with [CircuitPython](https://circuitpython.org/). It has already many key features out of the box:
```

Comment:
Useful update. It is confusing that LilyGo calls it the T8 and T-Display S2 to find the correct repository

- https://github.com/Xinyuan-LilyGO/LilyGo-T-Display-S2
- https://github.com/Xinyuan-LilyGO/LilyGO-T-DisplayGD32
- https://github.com/Xinyuan-LilyGO/LILYGO-T-display-RP2040
- https://github.com/Xinyuan-LilyGO/TTGO-T-Display
- https://github.com/Xinyuan-LilyGO/LILYGO-T8
- https://github.com/Xinyuan-LilyGO/ESP32_S2